pkg/processor/procbuiltin/unwrap.go

```diff
@@ -178,11 +179,15 @@ func (o *openCDCUnwrapper) UnwrapKey(structData record.StructuredData) (record.D
 	case record.Data:
 		key = k
 	case string:
```

Comment:
We should take care of structured keys as well.

Nitpick: maybe it's good to extract the code from UnwrapPayload into a method, and just re-use it here.